types/babel__traverse/babel__traverse-tests.ts

```diff
@@ -20,11 +20,14 @@ const MyVisitor: Visitor = {
 const MyVisitor2: Visitor = {
     Identifier(path) {
         path.type; // $ExpectType "Identifier"
-        path.parentPath; // $ExpectType NodePath<Node>
+        path.parentPath; // $ExpectAssignable NodePath<Node> | null
```

Comment:
I think this was to cover `File` not having a parent and hence being `null` (rather than `NodePath<null>`!). I'll figure something out to narrow this further.